rome_app/templates/rome_templates/essay_detail.html

```diff
@@ -195,19 +214,25 @@
 <h3>Books and Prints</h3>
 <div id="relatedworks">
 {% for work in related_list %}
-    <div class="work">
+   
+    {% if work.ppid %}
```

Comment:
Looks like you don't want to render anything if work.ppid is True: you could change this to just {% if not work.ppid %} and not use the else.